# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/koules/gcc_parser_fix.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -pruN koules-cvs-2003-04-16-orig/koules.c koules-cvs-2003-04-16/koules.c
--- koules-cvs-2003-04-16-orig/koules.c	2003-07-31 07:33:29.000000000 +0200
+++ koules-cvs-2003-04-16/koules.c	2003-07-31 07:28:35.000000000 +0200
@@ -1213,8 +1213,8 @@ creators_points (int radius, int x1, int
 	r=sqrt((double)((x-midX)*(x-midX) + (y-midY)*(y-midY)));
 
 /* fix by Alexey Bokov 25-06-98 */
-//        r=(r*radius/r1)/r*0.9; // why do we multyply by r then divide by r?
-				 // causing SIGFPE when r==0...
+/*        r=(r*radius/r1)/r*0.9; // why do we multyply by r then divide by r?
+				 // causing SIGFPE when r==0... */
 	r=(radius/r1)*0.9;
 	x2=x1+(x-midX)*r;
 	y2=y1+(y-midY)*r;
